components/automate-chef-io/content/docs/api-tokens.md

```diff
@@ -10,124 +10,48 @@ toc = true
-Standard tokens have much more limited permissions; they are designed for Chef Clients and InSpec Agents to send data to Chef Automate.
-You can also use a standard token to access any part of the Chef Automate API if you write a policy granting it specific access. However, granting a user access to the ``auth:tokens`` resource also gives that user access to the administrative token.
-
-Before you follow these instructions, we recommend you install the JSON processor [jq](https://stedolan.github.io/jq/) to ensure readable output. Without it, some commands may need to be modified.
-
-## Viewing API Tokens
-
-![API Tokens](/images/docs/admin-tab-API-tokens-list.png)
-
-From the **Settings** tab, navigate to the _API Tokens_ page. Admin users and non-admin users with admin-like authorization defined in policies, will see all of the tokens registered in Chef Automate.
-
-For more information on defining policies, see [IAM v1 policies]({{< relref "iam-v1-policies.md" >}}).
+Permission on the `iam:tokens` resource is required to interact with tokens. Any user that is part of the `admins` team, or the `Administrator` policy will have this permission, otherwise, [IAM v2 custom policies]({{< relref "iam-v2-guide.md#creating-custom-policies" >}}) can be created to assign this permission.
 
 ## Managing API Tokens
 
-### The `admin-token` and Global API Access
-
-Create the Admin API token granting global API access from the command line. Follow the steps in [Creating an Admin API token]({{< relref "#creating-an-admin-api-token" >}}). If you've already created an Admin token, you will see it in the API token list.
-
-## Creating a Standard API Token
+### Creating API Tokens
+Navigate to _API Tokens_ in the **Settings** tab. Then use the **Create Token** button, which opens a helper window for entering the token's _name_, a unique _ID_, and optionally assign the API token to some _Projects_. Token IDs can contain lower-case letters, hyphens, underscores, and numbers, but they cannot contain symbols or upper-case letters.
 
-Create standard API tokens with limited API access levels, or for use with Chef Client and Inspec data reporting, using either the user interface or the [API]({{< relref "api-tokens.md#creating-a-standard-api-token" >}}).
-By default, standard tokens have access only to data collection API endpoints, so to use them outside the context of Chef Clients and InSpec Agents, you'll need to [create a policy]({{< relref "iam-v1-overview.md#default-authorization-settings" >}}) for your token.
+After a token is created from the UI it will have no permissions, to assign the token permissions navigate to _Policies_ in the **Settings** tab, locate the appropriate policy and then add the token as a member of the policy using a [member expression]({{< relref "iam-v2-guide.md#member-expressions" >}}).
```

Comment:
```suggestion
After a token is created from the UI it will have no permissions. To assign the token permissions, navigate to _Policies_ in the **Settings** tab, locate the appropriate policy, and then add the token as a member of the policy using a [member expression]({{< relref "iam-v2-guide.md#member-expressions" >}}).
```